src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -154,414 +175,872 @@ module TaskSeq =
     val tail: source: taskSeq<'T> -> Task<taskSeq<'T>>
 
     /// <summary>
-    /// Returns the last element of the task sequence from <paramref name="source" />, or <see cref="None" /> if the sequence is empty.
+    /// Returns the last element of the input task sequence given by <paramref name="source" />,
+    /// or <see cref="None" /> if the sequence is empty.
     /// </summary>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The last element of the task sequence, or None.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
     val tryLast: source: taskSeq<'T> -> Task<'T option>
 
     /// <summary>
-    /// Returns the last element of the <see cref="taskSeq" />.
+    /// Returns the last element of the input task sequence given by <paramref name="source" />.
     /// </summary>
-    /// <exception cref="ArgumentException">Thrown when the sequence is empty.</exception>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The last element of the task sequence.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
+    /// <exception cref="T:ArgumentException">Thrown when the task sequence is empty.</exception>
     val last: source: taskSeq<'T> -> Task<'T>
 
     /// <summary>
-    /// Returns the nth element of the <see cref="taskSeq" />, or <see cref="None" /> if the sequence
-    /// does not contain enough elements, or if <paramref name="index" /> is negative.
-    /// Parameter <paramref name="index" /> is zero-based, that is, the value 0 returns the first element.
+    /// Returns the nth element of the input task sequence given by <paramref name="source" />,
+    /// or <see cref="None" /> if the sequence does not contain enough elements, or <paramref name="index" />
+    /// is negative.
+    /// The index is zero-based, that is, using index 0 returns the first element.
     /// </summary>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The nth element of the task sequence, or None if it doesn't exist.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
     val tryItem: index: int -> source: taskSeq<'T> -> Task<'T option>
 
     /// <summary>
-    /// Returns the nth element of the <see cref="taskSeq" />, or raises an exception if the sequence
-    /// does not contain enough elements, or if <paramref name="index" /> is negative.
+    /// Returns the nth element of the input task sequence given by <paramref name="source" />,
+    /// or raises an exception if the sequence does not contain enough elements, or <paramref name="index" />
+    /// is negative.
+    /// The index is zero-based, that is, using index 0 returns the first element.
```

Comment:
no harm I guess, but you dont go to the troubel of mentioning this for the iteri and friends. Perhaps a "(0-based)" in the '<param` might be enough?

Reply:
Can you continue in #136 instead @bartelink? It was my mistake to base this branch off of that one.

Reply:
I don't think you see my comments, lol

Reply:
I think this was taken from the `remarks` in the original `seq` docs. But yes, it should be in `iteri` and friends as wel.

Reply:
If the VS tooling was better, than yes, mentioning it in the param was enough. But since it isn't, I need to add all this info in the tooltip main bulb.

Reply:
Updated it for all functions to be clearer.